### PR TITLE
refactor Introduction to Primitives

### PR DESCRIPTION
**Pull request by abbycross**
Closes https://github.com/Qiskit/documentation/issues/34

### CODE REVIEW

**Comment by javabster** [comment]
Sorry for the delay, ultimately the decision we reached is this:

- the introduction to primitives page should be completely removed\
- the content in this PR should instead be added to the new "What is Qiskit?" page, issue tracked here: https://github.com/Qiskit/documentation/issues/30

**Comment by abbycross** [comment]
Note that the material about open-source primitives vs. Runtime primitives is being removed in this PR; need to work it back in if deemed necessary @javabster (see lines 44-46 [on the left](https://github.com/Qiskit/documentation/pull/84/files))

File: docs/build/primitives.mdx [hunk 1]
```diff
@@ -1,68 +1,45 @@
...
-
-A probability distribution is often of interest in optimization problems that return a classical bit string, encoding a certain solution to a problem at hand. In these cases, you might be interested in finding a bit string that corresponds to a ket value with the largest probability of being measured from a quantum state, for example.
-
-An expectation value of an observable could be the target quantity in scenarios where knowing a quantum state is not relevant. This often occurs in optimization problems or chemistry applications. For example, when trying to discover the extremal energy of a system.
-
-## What is the difference between the open-source primitives and primitives available through Qiskit Runtime?
-
-The open-source primitives contain the base classes (to define interfaces) and a reference implementation. The Qiskit Runtime primitives provide a more sophisticated implementation (such as with error mitigation) as a cloud-based service.
+## How to use Qiskit primitives
```
Comment (Eric-Arellano) [inline docs/build/primitives.mdx:28]: Note that this content used to be a dedicated FAQ page in qiskit-ibm-runtime and now won't exist: https://github.com/Qiskit/documentation/pull/123#issuecomment-1753539345

**Comment by javabster** [comment]
Can we revive this PR to address the Jay feedback we got that's tracked in this issue: https://github.com/Qiskit/documentation/issues/254

**Comment by abbycross** [comment]
Reviving and moving this work to #293, so closing this PR.

**Pull request closed without merging**